source/conf.py

```diff
@@ -63,40 +63,10 @@ def setup(app):
 # Redirects using: https://pypi.org/project/sphinx-reredirects/
 redirects = {
      "integrations/jira": "https://mattermost.gitbook.io/plugin-jira/",
-     "process/help-wanted": "https://handbook.mattermost.com/contributors/contributors/help-wanted",
-     "process/sg_rest_markup": "https://handbook.mattermost.com/operations/operations/company-processes/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines/documentation-style-guide#document-structure",
-     "process/security-release": "https://handbook.mattermost.com/operations/research-and-development/product/release-process/security-release",
-     "process/community-overview": "https://handbook.mattermost.com/contributors/contributors/community",
-     "process/handbook": "https://handbook.mattermost.com/company/about-mattermost",
-     "process/release-faq": "https://handbook.mattermost.com/operations/research-and-development/product/release-process/release-overview",
+     
```

Comment:
We can likely also add code-only comments to split out sections prefaced with ``..  comment goes here``